webpack/webpack.dev.js

```diff
@@ -1,9 +1,26 @@
+const path = require('path');
 const merge = require('webpack-merge');
 const common = require('./webpack.common');
 
 module.exports = merge(common, {
   mode: 'development',
 
+  devtool: 'eval-source-map',
+
+  output: {
+    filename: '[name].js',
+    chunkFilename: '[name].chunk.js'
+  },
+
+  devServer: {
+    contentBase: path.resolve(__dirname, '../public/index.html'),
+    port: 3000,
+    open: true,
```

Comment:
@vinhlee95 we've already set 'open' here, so no need to add it to script

Reply:
Nice